Formula/pipgrip.rb

```diff
@@ -47,10 +49,17 @@ class Pipgrip < Formula
   end
 
   def install
-    virtualenv_install_with_resources
+    venv = virtualenv_create(libexec, Formula["python@3.8"].opt_bin/"python3")
+    venv.pip_install resources
+    venv.pip_install buildpath
+
+    gcc_path = Formula["gcc"].bin
+    (bin/"pipgrip").write_env_script(libexec/"bin/pipgrip", { CC: gcc_path/"gcc-10", CXX: gcc_path/"g++-10" })
```

Comment:
Plain gcc does not work?

Reply:
Not sure exactly what you mean here

Reply:
The binary name. It's gcc-10 now, could this be just gcc?

Reply:
```console
$ ls /usr/local/opt/gcc/bin
c++-10                              gfortran-10
cpp-10                              lto-dump-10
g++-10                              x86_64-apple-darwin19-c++-10
gcc-10                              x86_64-apple-darwin19-g++-10
gcc-ar-10                           x86_64-apple-darwin19-gcc-10
gcc-nm-10                           x86_64-apple-darwin19-gcc-10.1.0
gcc-ranlib-10                       x86_64-apple-darwin19-gcc-ar-10
gcov-10                             x86_64-apple-darwin19-gcc-nm-10
gcov-dump-10                        x86_64-apple-darwin19-gcc-ranlib-10
gcov-tool-10                        x86_64-apple-darwin19-gfortran-10
gfortran
```

I don't see a non-versioned option here

Reply:
I could maybe do something like:

```ruby
gcc_version = Formula["gcc"].version.to_s.split(".").first

(bin/"pipgrip").write_env_script(libexec/"bin/pipgrip", { CC: gcc_path/"gcc-#{gcc_version}", CXX: gcc_path/"g++-#{gcc_version}" })
```

Reply:
Could be like that